crates/bermuda/src/lib.rs

```diff
@@ -194,6 +194,14 @@ fn triangulate_polygons_with_edge(
 ) -> PyPolygonTriangulation {
     // Convert the numpy array into a rust compatible representation which is a vector of points.
     let polygons_ = numpy_polygons_to_rust_polygons(polygons);
+    if polygons_.len() == 1 && is_convex(&polygons_[0]) {
```

Comment:
Might be helpful to state that this is operating on the first polygon in the vector. I'm assuming that is what is happening.

Reply:
Added comment, could you check?